components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -626,4 +626,15 @@ public static class AutoLoginConstant {
         public static final String CONTENT = "content";
         public static final String DOMAIN = "domain";
     }
+
+    /**
+     * Define logging constants for framework.
+     */
+    public static class LogConstants {
+
+        public static final String AUTHENTICATION_FRAMEWORK = "authentication-framework";
+        public static final String FAILED = "FAILED";
+        public static final String SUCCESS = "SUCCESS";
```

Comment:
Aren't these constants defined somewhere already?

Reply:
We have defined similar LogConstants in the oauth repo as well. 
So far, we have defined them component-wise. Not in a root-level repo

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/4263/commits/ecab44c9e838c47a16f0b98beadde1dc03c7d291